src/App.js

```diff
@@ -1,9 +1,28 @@
 import React from 'react';
+import {Login} from "./Login";
+import {Register} from "./Register";
+import {Dashboard} from "./Dashboard";
+import {BrowserRouter as Router, Route, Switch, Redirect} from "react-router-dom"
+
+export const loginPath = '/login';
+export const registerPath = '/register';
 
 function App() {
+    const auth = localStorage.getItem("auth");
```

Comment:
It works on the opening of the page,

But when you save token on the Login page (event it is just a 'ok' string now), `App` component knows nothing about it, and the user will be redirected to dashboard only after refreshing the page.

The simplest solution is to add state for the token (smth like `const [auth, setAuth] = useState(localStorage.getItem("auth"))`) and pass the function `setAuth` to the Login page. After that on the login page save the token not only in the localStorage but also in the `App` compoennt.